From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Kleinschmidt <jkleinsc@electronjs.org>
Date: Tue, 2 Dec 2025 13:19:50 -0500
Subject: chore: disable set_appcontainer_acls.py

Setting the LPAC ACLS is causing issues when running on our Windows 2022
CI servers, so disable this step.

diff --git a/testing/scripts/common.py b/testing/scripts/common.py
index 208b0d1deffd97dc044c70c0c837f2401b419620..2b588f8c15e39d5dc8fbe170accceac5d13a0d7a 100644
--- a/testing/scripts/common.py
+++ b/testing/scripts/common.py
@@ -62,7 +62,7 @@ CORRECT_ACL_VARIANTS = [
 
 def set_lpac_acls(acl_dir, is_test_script=False):
   """Sets LPAC ACLs on a directory. Windows 10 only."""
-  if platform.release() != '10':
+  if '11' != '10':
     return
   try:
     existing_acls = subprocess.check_output(['icacls', acl_dir],
